nimare/tests/new_json_format/dset_study1.json

```diff
@@ -0,0 +1,75 @@
+{
+  "Studies": [
+    {
+      "Groups": [
+        {
+          "@id": "gg_id1",
+          "Name": "group1",
+          "N": 12
+        }
+      ],
+      "Conditions": [
+        {
+          "@id": "cc_id1",
+          "Name": "household items"
+        },
+        {
+          "@id": "cc_id2",
+          "Name": "animals"
+        }
+      ],
+      "@id": 9593960,
```

Comment:
I think the @id fields will eventually be populated by the central clearinghouse, and shouldn't be user-assigned. For objects that we think need some user-provided identifier, we should introduce explicit fields for that. E.g., schema.org provides an `identifier` property at the `Thing` level, and the working definition of `Point` includes a `label`. For the examples, I think we can just drop all `@id` properties.

Reply:
I had imagined these examples as prototypes of the completed json including fields that would not be entered by a human (such as @id).